index.html

```diff
@@ -0,0 +1,161 @@
+        <li class="nav-item"><a href="#index.html" class="live nav-link">Register</a></li>
+      </ul>
+
+
+    </nav>
+  </header>
+
+  <section class="about" id="about">
+    <div class="container">
+      <div class="topText">
+        <h3>"Hello! Tech world"</h3>
+      </div>
+      <div class="title">
+        <h1>GLOBAL-TECH</h1>
+        <h1>HACKATHON</h1>
+        <h1>2022</h1>
+      </div>
+      <div class="description">
+        <p>
+          The GT hackerthon aims to bring in startups, developers, strategists, and thinkers to explore Digital
+          opportunities and help solve pressing problems faced by large corporates and startups in key industry sectors.
+        </p>
+      </div>
+      <div class="date">
+        <p>2022.11.18(FRI) ~ 20(SUN)</p>
+      </div>
+      <div class="venue">
+        <p>
+          @Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+        </p>
+      </div>
+    </div>
+
+  </section>
+
+  <section class="program" id="program">
+    <div class="top-tit">
+      <h2>Main Program</h2>
+    </div>
+    <div class="line"></div>
+    <div class="card-container">
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_01 1.png" alt="icon" /></div>
+        <div class="card-title">Workshops</div>
+        <div class="card-text">The Hackathon will offer Workshops and helpful mentor sessions to educate and
+          prepare participants</div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_02 1.png" alt="icon" /></div>
+        <div class="card-title">Network</div>
+        <div class="card-text">Connect with tech companies and other participants and so much more. </div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_03 1.png" alt="icon" /></div>
+        <div class="card-title">Challenge</div>
+        <div class="card-text">To test the technical competencies of young developers by offering them opportunities to
+          solve critical challenges</div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_02 1.png" alt="icon" /></div>
+        <div class="card-title">Innovation</div>
+        <div class="card-text">To boost a culture of innovation, idea sharing, effective collaboration and creativeness
+          driven by enthusiasm towards a shared vision. </div>
+      </div>
+      <div class="card">
+        <div class="icon"><img src="./pictures/program_icon_05 1.png" alt="icon" /></div>
+        <div class="card-title">Prizes</div>
+        <div class="card-text">Shared $3000 for top 3 teams</div>
+      </div>
+    </div>
+    <div class="show_all_program">
+      <a href="#" class="web">View all programs>></a>
+    </div>
+    <div>
+      <button class="register"> Register</button>
+    </div>
+  </section>
```

Comment:
- [ ] In the program section, there are 2 vertical lines that are not part of the design. Kindly remove them 👍 

![Animation](https://user-images.githubusercontent.com/50721479/198384576-6d2b2add-8203-4099-a65e-23b2b0a6536e.gif)

- [ ] Kindly try to follow the design guidelines as much as possible for this section. This includes the size, the margins and the button 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198384937-09d02634-2c56-435f-905d-3eda5b605638.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198384905-feb3bcc6-02bc-4e31-8b7f-5f09c2c34b23.png)